lib/matplotlib/artist.py

```diff
@@ -11,22 +11,6 @@
 from .path import Path
 from .transforms import (Bbox, IdentityTransform, Transform, TransformedBbox,
                          TransformedPatchPath, TransformedPath)
-# Note, matplotlib artists use the doc strings for set and get
```

Comment:
that comment is essentially completely false now (even the magic string for aliases is "Alias for", not "alias for" :))